sdk/tracing/pom.service.xml

```diff
@@ -17,7 +17,7 @@
     <module>../storage/azure-storage-queue</module>
     <module>../storage/azure-storage-common</module>
     <module>../eventhubs/azure-messaging-eventhubs</module>
-    <module>../keyvault/azure-keyvault-secrets</module>
+    <module>../keyvault/azure-security-keyvault-secrets</module>
```

Comment:
@samvaity Are these module declarations in the tracing pom.service.xml still required based on your tracing PR?

Reply:
nope, they can be removed. 
I just merged the tracing-move PR to master. The rebase should take care of this.

Reply:
Actually, the PR didn't include to remove the tracing folder completely. 
@JonathanGiles we should be good to do that now?

Reply:
deleted the tracing folder